src/feedback/components/AmountPicker.vue

```diff
@@ -122,10 +133,32 @@ export default {
 </script>
 
 <style scoped lang="stylus">
+@import '~variables'
+
 .wrapper
   position relative
 
-.showOverlay .content
+.content
+  padding 30px 20px 5px
+  background hsl(0deg 0% 97%)
+  border-top-left-radius 13px
+  border-top-right-radius 13px
+
+.content-more-weight-info
+  height 30px
+  padding 0 20px
+  font-size 0.75rem
+  line-height 30px
+  text-align center
+  background hsl(0deg 0% 97%)
+  border-bottom-right-radius 13px
+  border-bottom-left-radius 13px
+
+  &.is-visible
+    background $info
+
+.showOverlay .content,
+.showOverlay .content-more-weight-info
```

Comment:
This is quite a lot of custom css here, where at least some of it is available via quasar classes, e.g. `text-center` (I think!), some margins, paddings, and maybe `bg-grey-<some number>`, and there is duplication which might be nice in a variable (the border radius stuff, I wonder if there are already some quasar variables for radius too?).

(in general we try and minimise the custom css and so far it was one of the least maintainable parts, and hard to reuse given we lack a coherent design system, beyond that which quasar provides)